README.md

```diff
@@ -176,6 +176,25 @@ usbDetect.stopMonitoring();
 
 Make sure you call `usbDetect.startMonitoring()` before any calls to `usbDetect.find()`.
 
+# Known problems:
+
+### If you are using this dependency with electron.js is posible that you see an error similar to: *detection.node was compiled against a different Node.js version using NODE_MODULE_VERSION 72. This version of Node.js requires NODE_MODULE_VERSION 80. Please try re-compiling or re-installing*
+
+You can solve this problem using the command below:
+```shell
+./node_modules/.bin/electron-rebuild
+```
+
+Steps:
+
+- If you have not installed electron-rebuild just install it with the command: `npm i -D electron-rebuild`
+- Remove from the `node-modules` folder the `usb-detection` folder.
+- Remove the file `packages-lock.json`
```

Comment:
typo: `package-lock.json`

Reply:
Why is it necessary to remove from the `package-lock.json`?

Reply:
Are exactly steps that I did do, but I have changed the redaction to indicate it as suggestion